bin/pfcmd-old.pl

```diff
@@ -75,7 +75,8 @@ =head1 SYNOPSIS
 use constant {
     INSTALL_DIR        => '/usr/local/pf',
     JUST_MANAGED       => 1,
-    INCLUDE_DEPENDS_ON => 2,
+    INCLUDE_START_DEPENDS_ON => 2,
+    INCLUDE_STOP_DEPENDS_ON => 3,
```

Comment:
These are flags not enumerations.
It should be changes to use bit notation.

0b0000001
0b0000010
0b0000100

To make it more clear